js/app.js

```diff
@@ -5,7 +5,6 @@
 
```

Comment:
The comments will be changed as I progress through the PRs, although I see your point, and it makes the review more difficult not having descriptive comments. I'll go ahead and add them.  

As for the copyright, it would be better to handle it in a separate PR to avoid blocking this one, since it's not clear what the TC's opinion is on this.